### PR TITLE
Properties: Show Predefined Name in Grid

### PR DESCRIPTION
**Pull request by blankse**
You select the properties by the predefined name but than you can't see it in the grid. Only the key and the description is there. I think the user should also see the name that he has selected.

### CODE REVIEW

File: bundles/AdminBundle/Resources/public/js/pimcore/element/properties.js [hunk 1]
```diff
@@ -270,6 +297,14 @@ pimcore.element.properties = Class.create({
                         renderer: Ext.util.Format.htmlEncode,
                         width: 230
                     },
+                    {
+                        text: t('name'),
+                        dataIndex: 'predefinedName',
```
Comment (dvesh3) [inline bundles/AdminBundle/Resources/public/js/pimcore/element/properties.js:302]: Should we display "Custom" here for custom properties?

Comment (dvesh3) [reply bundles/AdminBundle/Resources/public/js/pimcore/element/properties.js:302]: 🏓 @blankse

Comment (blankse) [reply bundles/AdminBundle/Resources/public/js/pimcore/element/properties.js:302]: @dvesh3 Good idea, I added it.

**Comment by dvesh3** [comment]
@blankse thanks!

**Pull request merged**